moped-editor/src/views/projects/projectView/ProjectComponents/mapSettings.js

```diff
@@ -45,29 +46,101 @@ export const mapParameters = {
   mapboxAccessToken: MAPBOX_TOKEN,
 };
 
+/*
+ * This configuration sets the mapStyle, sources, and layers associated with basemaps
+ * mapStyle becomes the value of the mapStyle prop of the react-map-gl map
+ * https://visgl.github.io/react-map-gl/docs/api-reference/map#mapstyle
```

Comment:
Do you want a period here after the first line? Even with that, the second line is unclear to me.

How's this: The `react-map-gl` map's prop `mapStyle` is used.

Reply:
Also, do you want this comment in the `TheMap` component js file?

Reply:
I totally agree that the wording is kind of wonky so I updated it to explicitly say that the k/v pair is used to pass a value to  the `mapStyle` prop of the map. Thanks!